components/style_traits/values.rs

```diff
@@ -450,7 +432,7 @@ impl_to_css_for_predefined_type!(::cssparser::RGBA);
 impl_to_css_for_predefined_type!(::cssparser::Color);
 impl_to_css_for_predefined_type!(::cssparser::UnicodeRange);
 
-#[macro_export]
+/// Define an enum type with unit variants that each corrsepond to a CSS keyword.
```

Comment:
tiny typo on "corrsepond"

```suggestion
/// Define an enum type with unit variants that each correspond to a CSS keyword.
```

Reply:
Applied, thanks!